actors/market/src/lib.rs

```diff
@@ -72,6 +77,21 @@ pub enum Method {
     OnMinerSectorsTerminate = 7,
     ComputeDataCommitment = 8,
     CronTick = 9,
+    // Method numbers derived from FRC-0042 standards
+    AddBalanceExported = frc42_dispatch::method_hash!("AddBalance"),
+    WithdrawBalanceExported = frc42_dispatch::method_hash!("WithdrawBalance"),
+    PublishStorageDealsExported = frc42_dispatch::method_hash!("PublishStorageDeals"),
```

Comment:
Flagging the same thing from a previous review that people will probably try to use this in the wrong way if it remains exported.  This can only be called by a control address of the provider SP.  This *can* be forced to work in user-contract accessible ways since control addresses can be evm actors but the constraints are weird and not good UX.

Reply:
I agree, but I think it's a result of the system being somewhat confusing (eg. clients not being able to PSDs today). I'd rather document the gotcha-UX and export the functionality, than keep it unexported for now.

One concern I have with our circumspect approach to exporting methods in this FIP is that we might be stifling innovation before it can begin. I agree that people are going to be confused by the behaviour here (even if we document it well), but my thought is that we need to hear that feedback, have a dialogue with users, and receive suggestions in order to drive the improvements we need.